source/Taboos/index.lagda

```diff
@@ -12,5 +12,6 @@ import Taboos.LLPO
 import Taboos.LPO
 import Taboos.WLPO
 import Taboos.P2
+import Taboos.FiniteSubsetTaboo    -- by Ayberk Tosun
```

Comment:
Import in alphabetical order in the index file.

Reply:
Done. Shall I also make a note of this convention in `CONTRIBUTING.md`?

Reply:
Only for index files. For other files, people occasionally may want to break this rule (although in most cases sorting would be preferable).